test_communication/test/test_messages_c.cpp

```diff
@@ -1126,3 +1127,108 @@ TEST_F(CLASSNAME(TestMessagesFixture, RMW_IMPLEMENTATION), test_boundedarraynest
   test_message_type<test_communication__msg__BoundedArrayNested>("test_boundedarraynested",
     ts);
 }
+
+template<>
+size_t get_message_num(test_communication__msg__DynamicArrayPrimitivesNested * msg)
+{
+  (void)msg;
+  return 1;
+}
+
+
+template<>
+void init_message(test_communication__msg__DynamicArrayPrimitivesNested * msg)
+{
+  test_communication__msg__DynamicArrayPrimitivesNested__init(msg);
+}
+
+template<>
+void get_message(test_communication__msg__DynamicArrayPrimitivesNested * msg, size_t msg_num)
+{
+  test_communication__msg__DynamicArrayPrimitives submsg;
+  const size_t size = get_message_num(&submsg);
+  test_communication__msg__DynamicArrayPrimitivesNested__init(msg);
+  test_communication__msg__DynamicArrayPrimitives__Array__init(&msg->msgs, size);
+  switch (msg_num) {
+    case 0:
+      for (size_t i = 0; i < size; ++i) {
+        get_message(&msg->msgs.data[i], i);
+      }
+      break;
+  }
+}
+
+template<>
+void verify_message(test_communication__msg__DynamicArrayPrimitivesNested & message, size_t msg_num)
```

Comment:
Can this function somehow call `verify_message(test_communication__msg__DynamicArrayPrimitives & message, size_t msg_num)` to avoid the need to explicitly check all the fields?